index.js

```diff
@@ -10,6 +10,10 @@ function isRelativeUrl(url) {
 }
 
 function sanitizeUrl(url) {
+  if (!url) {
```

Comment:
I'm unsure about this behavior. If we're doing this, would it make more sense for anything that's not a `string` to be sanitized to `about:blank`?

```js
if (!url || typeof url !== 'string') {
```

Reply:
maybe we just go all the way and migrate the entire codebase to typescript? 🤔

Reply:
TBH, I think the question here is "should this module care about this check? or should it assume a non-blank string input when it is being used?"

Reply:
- At the very least I'm thinking that the function shouldn't _break_ if passed a null string.
- Maybe it can explicitly throw an error instead

And I would agree that this change would necessitate major version bump.

Reply:
as in if a null string is passed in then clearly something has gone wrong in the calling function, and an error should be thrown. I agree, passing `about:blank` possibly allows that error to be hidden.

Reply:
Right now this is the error seen:
```
main-17329e71f6ca75b7a6c5-chunktemp.js:1 TypeError: Cannot read property 'replace' of undefined
    at sanitizeUrl (vendors~main-8d24e3a6bc8c08ce4fbd-chunktemp.js:13)
    at p.push (main-17329e71f6ca75b7a6c5-chunktemp.js:1)
```